### PR TITLE
Added custom devise mailer

### PR DESCRIPTION
**Pull request by egonm12**

### CODE REVIEW

File: app/mailers/custom_devise_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class CustomDeviseMailer < Devise::Mailer
+  helper :application # gives access to all helpers defined within `application_helper`.
+  include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
+  default template_path: 'devise/mailer' # to make sure that your mailer uses the devise views
+  layout 'mailer'
+
+  def confirmation_instructions(record, token, opts={})
+    add_logo_attachment
```
Comment (ariejan) [inline app/mailers/custom_devise_mailer.rb]: Maybe add another `before_action` to this custom mailer, like you did in `ApplicationMailer`?

File: app/mailers/custom_devise_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class CustomDeviseMailer < Devise::Mailer
```
Comment (ariejan) [inline app/mailers/custom_devise_mailer.rb]: Maybe name it differently? _Custom_ what? Why is it custom? The name does not reveal anything.

File: spec/mailers/custom_devise_mailer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe CustomDeviseMailer, type: :mailer do
+  context 'new user' do
+    let(:user) { User.create name: 'John Doe', email:'johndoe@example.com' }
+    let(:mail) {described_class.confirmation_instructions(user, "faketoken", {})}
+
+    # it 'renders the subject' do
```
Comment (ariejan) [inline spec/mailers/custom_devise_mailer_spec.rb]: Re-enable this test?

File: spec/mailers/custom_devise_mailer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe CustomDeviseMailer, type: :mailer do
```
Comment (ariejan) [inline spec/mailers/custom_devise_mailer_spec.rb]: Shouldn't there be a test to check if that logo is available / included?

File: app/mailers/custom_devise_mailer.rb (hunk 1 above)
Comment (egonm12) [reply app/mailers/custom_devise_mailer.rb]: Discussed and not possible. Resolved.

**Pull request merged**